etc/deploy/aws.sh

```diff
@@ -72,13 +85,33 @@ create_s3_bucket() {
   else
     aws s3api create-bucket --bucket ${BUCKET} --region ${AWS_REGION} --create-bucket-configuration LocationConstraint=${AWS_REGION}
   fi
+
+  if [ "$2" != "false" ]; then
```

Comment:
Could you add a short comment to the top of `create_s3_bucket` indicating that it takes two arguments, and what the second argument means? (the first is the name of the s3 bucket being created)